client_test/helpers_test.go

```diff
@@ -182,7 +337,7 @@ func checkPaymentChannel(t *testing.T, id types.Destination, o outcome.Exit, sta
 			t.Fatal(err)
 		}
 		if diff := cmp.Diff(expected, ledger, cmp.AllowUnexported(big.Int{})); diff != "" {
-			t.Fatalf("Payment channel diff mismatch (-want +got):\n%s", diff)
+			panic(fmt.Errorf("payment channel diff mismatch (-want +got):\n%s", diff))
```

Comment:
Happy to revert this change, but with the `panic` I was able to get a full stack trace and figure out which call to `checkPaymentChannel` was failing. With `t.Fatalf` I don't think we get any information about who called `checkPaymentChannel`